app/component/map/tile-layer/TileLayerContainer.js

```diff
@@ -371,42 +371,25 @@ class TileLayerContainer extends GridLayer {
         } else if (
           this.state.selectableTargets[0].layer === 'dynamicParkingLots'
         ) {
+          const { lat, lng } = this.state.coords;
           const {
-            free,
-            total,
-            state,
-            freeDisabled,
-            totalDisabled,
-            detailsUrl,
-            openingHours,
+            id: vehicleParkingId,
             name,
-            note,
-            lotType,
           } = this.state.selectableTargets[0].feature.properties;
-          const { lat, lng } = this.state.coords;
           const params = pickBy(
             {
               lat,
```

Comment:
Why do you need to pass lat, lng and name?

Reply:
![image](https://user-images.githubusercontent.com/21622725/123597806-15a41900-d7f4-11eb-9cd1-cbcc93fee0cb.png)
It's needed for showing this popup on the map. It's used in `SidebarMap.js`. I could also refactor that component to receive the required information via graphql, however, I don't think it's necessary because the location and the name won't change, so passing this information via url is okay.